src/common/plugin/types.ts

```diff
@@ -79,6 +81,11 @@ export interface EngineInfo {
   ) => IProcessorTransaction
 }
 
+export interface ServerConfig {
+  type: 'blockbook-nownode'
+  uris: string[]
+}
```

Comment:
This structure seems weird. Why not put `fallbackNownodeServers?: string[]` straight into the engine info / init info?

I mean, it's always possible to invent your own custom object type using arrays, like `type FakeObject<T> = Array<{ key: string, value: T}>`, but this sort of thing is rarely a good idea. Just use normal objects.

The `ServerConfig` type seems to be implementing this anti-pattern, with `type` as the key and `uris` as the value.

Reply:
Because: It's easier to enumerate nominal types then fields in typescript.

Reply:
Ok, we discussed this design some more, and it makes sense if the servers need extra data, such as contract addresses or such. In that case, it might be `{ type: 'foo', servers: [], queryParams: '' }`, and so we want the flexibility of the array approach. This will impact the way the info server assembles the core rollup payload, so we might need a follow-up PR over there.